### PR TITLE
feat(matrix): update support on PostgreSQL + Ubuntu

### PR DESCRIPTION
**Pull request by educhastenier**
Relates to https://bonitasoft.atlassian.net/browse/PROD-367
and https://bonitasoft.atlassian.net/browse/PROD-365

### CODE REVIEW

File: modules/runtime/pages/hardware-and-software-requirements.adoc [hunk 1]
```diff
@@ -75,7 +75,7 @@ Software required for Bonita Platform.
 | (see note 2)
 
 | PostgreSQL
-| 15.3 and higher in the 15.x line
+| 16.4 and higher in the 15.x line
```
Comment (rbioteau) [inline modules/runtime/pages/hardware-and-software-requirements.adoc]:
```suggestion
| 16.4 and higher in the 16.x line
```

File: modules/ROOT/pages/release-notes.adoc [hunk 2]
```diff
@@ -32,6 +32,17 @@ xref:api:engine-api-overview.adoc[Engine APIs] that allowed applications manipul
 
 xref:api:rest-api-overview.adoc[REST APIs] (see _Application_ section) now also return application links in addition of legacy applications.
 
+
+== Updates
```
Comment (educhastenier) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
== Support matrix changes
```

**Review by rbioteau — CHANGES_REQUESTED** [review]
update docker image version in examples

File: modules/ROOT/pages/release-notes.adoc [hunk 3]
```diff
@@ -32,6 +32,17 @@ xref:api:engine-api-overview.adoc[Engine APIs] that allowed applications manipul
...
+
+=== PostgreSQL
```
Comment (educhastenier) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
=== PostgreSQL update
```

File: modules/ROOT/pages/release-notes.adoc [hunk 4]
```diff
@@ -32,6 +32,17 @@ xref:api:engine-api-overview.adoc[Engine APIs] that allowed applications manipul
...
+
+Support for PostgreSQL has been updated to version **16.4**. See xref:runtime:hardware-and-software-requirements.adoc[hardware and software requirements] for the full matrix support.
+
+=== Ubuntu
```
Comment (educhastenier) [inline modules/ROOT/pages/release-notes.adoc]:
```suggestion
=== Ubuntu update
```

**Review by rbioteau — APPROVED** [review]

**Comment by educhastenier** [comment]
> update docker image version in examples

We already use `latest`

**Pull request merged**